extensions/amp-story/0.1/variable-service.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {StateChangeType} from './navigation-state';
+import {dev} from '../../../src/log';
+
+
+/**
+ * Variable service for amp-story.
+ * Used for URL replacement service. See usage in src/url-replacements-impl.
+ */
+export class VariableService {
```

Comment:
Nit: How about `AmpStoryVariableSource`? There's already a `VariableService` in amp-analytics.

Reply:
Meeting you halfway by naming it `AmpStoryVariableService`. (see above)